lib/pbio/drv/legodev/legodev_pup.c

```diff
@@ -48,23 +44,37 @@ typedef enum {
 
 // Device connection manager state for each port
 typedef struct {
-    dev_id_group_t dev_id1_group;
+    /** Most recent one-off device ID candidate. */
```

Comment:
I'm not sure what `one-off` means in this context. Perhaps there is a better word choice?